convert/jenkinsjson/json/publishHTML.go

```diff
@@ -0,0 +1,53 @@
+package json
+
+import (
+	"fmt"
+	harness "github.com/drone/spec/dist/go"
+)
+
+func ConvertPublishHtml(node Node, variables map[string]string) *harness.Step {
+
+	s, err := ToJsonStringFromStruct[Node](node)
+	if err != nil {
+		return &harness.Step{}
+	}
+	publishHtmlParameterMap, err := ToStructFromJsonString[PublishHtmlParameterMap](s)
+	if err != nil {
+		fmt.Println(err)
+	}
+	pmt := publishHtmlParameterMap.ParameterMap.Target
+
+	step := &harness.Step{
+		Id:   SanitizeForId("UploadPublish", node.SpanId),
+		Name: "Upload and Publish",
+		Type: "plugin",
+		Spec: &harness.StepPlugin{
+			Connector: "<+input>",
+			Image:     DroneS3UploadPublish,
+			With: map[string]interface{}{
+				"aws_access_key_id":     "<+input>",
+				"aws_secret_access_key": "<+input>",
+				"aws_bucket":            "<+input>",
+				"aws_default_region":    "<+input>",
+				"source":                pmt.ReportDir,
+				"target":                "<+pipeline.sequenceId>",
```

Comment:
This will conflict across pipelines. Shouldn't we use `pipelineId/sequenceId` ?
@Ompragash @senthilhns